### ISSUE TITLE
Remaning Neighborhood tasks

### ISSUE THREAD

**Issue by abettermap** [Language-Mapping/language-map#225]
(from #222)

- [x] move toggle out of Search Locations into /Explore/Neighborhood and one level deeper
- [x] for /Explore/Neighborhood/Something with no primary langs (???), rm the preceding text before where the cards would normally be.
- [x] support intro text from summary field in AT
- [x] show borough as subtitle in /Explore/Neighb/Something

## Later
(moving to new issue)

- support auto zoom on initial page load when starting from something like /Explore/Neighborhood/Astoria.
- Share btns if we have to (???). Was my idea in the SOW, but we've also added 1k tasks to it, so my vote is making it optional in favor of the other massive tasks i created for myself. Not to mention it's kinda useless unless i fix the auto-zoom on load.

**Comment by abettermap** [comment]
@rperlin-ela as far as next steps/priorities go, after #217 is merged, what are we leaning towards? some big focuses, so let's pick one:

1. the issue above, minus "support auto zoom on initial page load", which i don't think i'll be able to fix, at least not without burning another day literally just for that edge case. if i don't fix though, that means we shouldn't include Share btns in /Explore/Neighb/Something like i suggested. got bigger fish to fry anyway!
2. abstracting/refactoring and genericizing the neighbs code in preparation for counties and census
3. tooltips on hover for points. FYI could lose a day on this as well. i'm not doing this for polygons, sorry. i think all the other non-SOW things i've added should make up for what might have been a potentially annoying addition (for polygons, not points) anyway.
4. other little minor fixes and tweaks that keep popping up

i could certainly do # 2 while the data is being prepped, but it might be easier once it's ready. 

# 1 is probably the most gratifying and directly applicable to neighborhoods, and i'm terrified of # 3 after the major related defeats i faced in the last couple days, and # 4 seems better for later in the week.

so # 1 i guess is my vote, but any thoughts?

**Comment by rperlin-ela** [comment]
No talk of defeat. I vote for 1 also. Yeah, scrap the share btns.

**Comment by abettermap** [comment]
Sweet, if I ever fix the on load issue, I can add the share. They are not difficult, but the auto zoom problem is.

Just to confirm, we don't need any toggle to enable/disable auto zoom in the explore panels for neighborhoods and counties, correct￼? I don't find it annoying in the least, especially when neighborhoods layer is not visible.￼

**Comment by rperlin-ela** [comment]
Nah I think it’s fine to auto-zoom people around

**Comment by abettermap** [comment]
FYI you've got at least one neighborhood w/trailing space:

![image](https://user-images.githubusercontent.com/4974087/113210621-aab56700-9231-11eb-81fb-0272ddf0266c.png)

**Comment by abettermap** [comment]
Unrelated, but THIS happened:

![image](https://user-images.githubusercontent.com/4974087/113221218-d5a7b700-9241-11eb-95fd-ddc43a3402ab.png)

which is a step up from current:

![image](https://user-images.githubusercontent.com/4974087/113220656-cb38ed80-9240-11eb-8a6b-30342e116904.png)

run with it??

**Comment by abettermap** [comment]
definitely better w/just 3 lines preview:

![image](https://user-images.githubusercontent.com/4974087/113221450-4b138780-9242-11eb-802c-dd5fd7aad64c.png)

**Comment by rperlin-ela** [comment]
Love it — inspires me to keep churning out the words!

**Comment by abettermap** [comment]
words are lookin good, keep em coming.

the toggle was easy to wire up:

![image](https://user-images.githubusercontent.com/4974087/113224263-494cc280-9248-11eb-8b28-2dcadd0612b6.png)

1. can we remove the "i"?
2. placement and size good enough?

**Comment by abettermap** [comment]
> Neighborhoods are shown within NYC's five boroughs, and counties (coming soon) for the surrounding areas. Source: NYC Census 2020 map

current the "i" popout has that. i'm not hatin' on the "i", just wondering if the text could go in your /Explore/Neighborhoods intro, or About or Help?

**Comment by rperlin-ela** [comment]
Good call, don’t need that “i”, will get the message across.

**Comment by abettermap** [comment]
axed. 

oh and here's how it's looking w/your summary:

![image](https://user-images.githubusercontent.com/4974087/113224920-b57bf600-9249-11eb-8fcf-6ef6d7e3f303.png)

and without any primaries or summary:

![image](https://user-images.githubusercontent.com/4974087/113225003-e2300d80-9249-11eb-8676-5855afe6f130.png)

sufficient?

i like it but only thing i might add is the "clear selection" btn. it's semi-disorienting and frustrating to not be able to jump ship. not sure where clear would take you, maybe just to /Explore/Neighborhoods? i know we can do that in like four different ways haha, but an escape hatch isn't terrible. can call it "Show all neighborhoods" or something, and can just be a link text instead of button.

how about after you answer the other things first, i push up what i have and you can play around to see what you think? the clear would be easy, just let me know
